gemrb/GUIScripts/pst/GUIOPT.py

```diff
@@ -388,26 +388,23 @@ def LoadGame ():
 def OpenQuitMsgWindow ():
 	global QuitMsgWindow
 
-	#GemRB.HideGUI()
-
-	if QuitMsgWindow:		
-		if QuitMsgWindow:
-			QuitMsgWindow.Unload ()
-		QuitMsgWindow = None
-		GemRB.SetVar ("FloatWindow", -1)
-		GemRB.SetVar ("AskAndExit", 0)
-		
-		#GemRB.UnhideGUI ()
+	if QuitMsgWindow:
 		return
-		
+
 	QuitMsgWindow = Window = GemRB.LoadWindow (4)
 	GemRB.SetVar ("FloatWindow", QuitMsgWindow.ID)
-	
+
 	# Save
 	Button = Window.GetControl (0)
 	Button.SetText (28645)
 	Button.SetEvent (IE_GUI_BUTTON_ON_PRESS, SaveGame)
 
+	EnteredGame = GemRB.GetSystemVariable(SV_ENTERED_GAME)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['SV_ENTERED_GAME' may be undefined, or defined from star imports: GUIDefines](https://app.codacy.com/app/gemrb/gemrb/pullRequest?prid=4591638)